packages/@dcl/ecs/src/composite/serialization.ts

```diff
@@ -0,0 +1,46 @@
+import { Entity } from '../engine/entity'
+import { Composite } from './types'
+
+/**
+ * @public
+ * @deprecated composite is not being supported so far, please do not use this feature
+ */
+export function compositeFromJson(jsonComposite: any): Composite {
```

Comment:
```ts
type CompositeJson  = {
  id?: OriginalCompoisteType['id'] | unknown
  components?: OriginalCompoisteType['components'] | unknown
}
```
```suggestion
export function compositeFromJson(jsonComposite: CompositeJson: Composite {
```
in case we have a typo or anything in the future :panic:

Reply:
this will be removed with another composite serialization